sync-themes.sh

```diff
@@ -0,0 +1,4 @@
+#!/bin/sh
```

Comment:
Can we put this in a Makefile, so that way we can just `make themes.json` after deleting it:

```
themes.json:
   ....
```

What do you think? If you prefer keeping a binary let's put this in a `bin/` folder.

Reply:
I can move into a bin or scripts folder, and also add a makefile... 

right now, you can also run it with `go generate` fwiw

Reply:
Ah if it's available with `go generate` let's delete this file then!

Reply:
(personally, think it's good to have only one way of doing things)

Reply:
done